daemon/kmd/wallet/driver/ledger.go

```diff
@@ -209,7 +209,24 @@ func (lw *LedgerWallet) SignTransaction(tx transactions.Transaction, pw []byte)
 
 // MultisigSignTransaction implements the Wallet interface.
 func (lw *LedgerWallet) MultisigSignTransaction(tx transactions.Transaction, pk crypto.PublicKey, partial crypto.MultisigSig, pw []byte) (crypto.MultisigSig, error) {
-	return crypto.MultisigSig{}, errNotSupported
+	isValidKey := false
+	sig, err := lw.signTransactionHelper(tx)
```

Comment:
Should this code first check that the key is present in the array, before it asks the user to page through the transaction on the Ledger's small screen and approve it?  I think that means two loops over `partial.Subsigs`, one before `signTransactionHelper` and one after, but loops are negligible compared to asking the user to sign a transaction that turns out to be useless because it's the wrong key.